react-native.config.js

```diff
@@ -6,6 +6,16 @@ module.exports = {
 			},
 			android: {
 				sourceDir: './platforms/android'
+			},
+			windows: {
+				sourceDir: './platforms/windows',
+				solutionFile: 'SQLitePlugin.sln',
+				projects: [
+				  {
+					projectFile: 'SQLitePlugin\\SQLitePlugin.vcxproj',
```

Comment:
The double backslashes cause this error when running `pod install` for iOS:
`[!] Invalid Podfile file: 767: unexpected token at 'Error: ENOENT: no such file or directory, open '/Users/astevens/code/goformz/mobile-react-native/node_modules/react-native-sqlite-storage/platforms/windows/SQLitePlugin\SQLitePlugin.vcxproj'`

Everything seems to work if they're replaced with a single forward slash

Reply:
Good catch! I updated the file.